v1/topdown/http_test.go

```diff
@@ -3295,105 +3295,59 @@ func TestInterQueryCacheValueClone(t *testing.T) {
 	}
 }
 
-func TestIntraQueryCache_ClientError(t *testing.T) {
+func TestQueryError(t *testing.T) {
```

Comment:
This is hopefully not the only test asserting proper error reporting for queries, so the new name might be a bot too general.

Reply:
I updated the name to `TestRaisingHTTPClientQueryError` to be more descriptive